package Exception2;

public class BasicWithFinally2_try 
{
	public static void main(String[] args) 
	{
		try
		{
		System.out.println("Highness");
		}
		catch(NullPointerException BasicWithFinally2_try )
		{
			System.out.println("Just");
		}
		finally
		{
			System.out.println("PJS");
		}
		
	}
}
